src/models/timeaxisheaderview.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "eventmodel.h"
 #include "filterandzoomstack.h"
+#include "util.h"
```

Comment:
../util.h otherwise this doesn't compile for me

Reply:
Correct. This has been fixed.